.github/workflows/deploy.yml

```diff
@@ -63,16 +63,16 @@ jobs:
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v3
         with:
-          hugo-version: '0.97.3'
+          hugo-version: '0.148.2'
```

Comment:
@nuclearcat I am trying to update hugo, but the workflow stills shows:

```
Run peaceiris/actions-hugo@v3
Hugo version: 0.97.3
```
https://github.com/kernelci/kernelci-project/actions/runs/16916278037/job/47930853147#step:11:6

Reply:
@nuclearcat Not sure what branch is that you referring to. This PR here has all the changes we need. Maybe you can push it to a branch in the main repo?